From dbf979098f82f0fabae4083445cd631ded96fa43 Mon Sep 17 00:00:00 2001
From: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
Date: Mon, 25 Nov 2019 15:26:18 +0800
Subject: [PATCH] arm64: dts: add armada-3720-db-B-double.dts

Add armada-3720-db-B-double.dts to set both eth0 and eth1 as kernel port.

Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 .../dts/marvell/armada-3720-db-B-double.dts    | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)
 create mode 100644 arch/arm64/boot/dts/marvell/armada-3720-db-B-double.dts

diff --git a/arch/arm64/boot/dts/marvell/armada-3720-db-B-double.dts b/arch/arm64/boot/dts/marvell/armada-3720-db-B-double.dts
new file mode 100644
index 000000000000..d0d77f6cd7b0
--- /dev/null
+++ b/arch/arm64/boot/dts/marvell/armada-3720-db-B-double.dts
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+/*
+ * Device Tree file for Marvell Armada 3720 development board
+ * This board file supports the B configuration of the board
+ * Where Eth1 and Eth0 are kernel-space
+ */
+
+#include "armada-3720-db-B.dts"
+
+/* Set Eth0 as Kernel port */
+&eth0 {
+	/delete-property/musdk-status;
+};
+
+/* Set Eth1 as Kernel port */
+&eth1 {
+	/delete-property/musdk-status;
+};
-- 
2.17.1

